include/minc_volume_io.h

```diff
@@ -52,9 +58,13 @@
 #ifdef Windows_NT
 #undef ERROR
 #endif // Windows_NT
+
 #include <volume_io.h> //from MNI
```

Comment:
It appears to be the case that the structs VIO_General_Transform and VIO_Transform are the only things used from volume_io.h.  I'm thinking it makes sense to just include these structs directly here in this file (maybe adapted a bit for remove the multiple embedded structs), and just not include volume_io.h at all.  this would sever the depedency completely, and would mean all the BEVIN_EXCLUDE_MINC conditionals could be removed (ie, returned to the original code).  Where the adapted structs are included, comments could be added to acknowledge the source of the adapted code (ie, the copyright notice at the top of geom_structs.h).

Reply:
Nick, should I accept this pull or wait until this is sorted out?